clients/team_allocation_component/src/team_allocation/pages/StudentSurvey/components/SkillRanking.tsx

```diff
@@ -0,0 +1,86 @@
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card'
+import { RadioGroup, RadioGroupItem } from '@/components/ui/radio-group'
+import { Separator } from '@/components/ui/separator'
+import { Star } from 'lucide-react'
+import { Skill } from 'src/team_allocation/interfaces/skill'
```

Comment:
Da stimmt glaube ich der Import nicht. 

```suggestion
import { Skill } from '../../../interfaces/skill'
```